### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

**Comment by pstorch** [comment]
Hi @Vistaus,

thank you very much for the dutch translation.

The CI build found a few errors in the strings.xml file:

```
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource compilation failed
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:57:3-61:14: AAPT: error: unescaped apostrophe in string
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:57:3-61:14: AAPT: error: not a valid string.
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:64:3-73:14: AAPT: error: unescaped apostrophe in string
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:64:3-73:14: AAPT: error: not a valid string.
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:18:3-95: AAPT: error: unescaped apostrophe in string
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:18:3-95: AAPT: error: not a valid string.
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:102:3-196: AAPT: error: unescaped apostrophe in string
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:102:3-196: AAPT: error: not a valid string.
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:214:5-73: AAPT: error: unescaped apostrophe in string
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-nl/strings.xml:214:5-73: AAPT: error: not a valid string.
         
     /home/travis/build/RailwayStations/RSAndroidApp/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-nl/values-nl.xml: AAPT: error: file failed to compile.
  ```

Can you have a look?

**Comment by Vistaus** [comment]
You're welcome :)

How do I figure out what's went wrong? 'Cause the TravisCI output doesn't really seem to point me to any specific strings.

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  <string name="count_of_images">Je hebt %1$d stationsfoto(\'s)</string>
+  <string name="search_hint">Station zoeken</string>
+  <string name="app_info_title">Over de app</string>
+  <string name="app_info_ok">Ok�</string>
+  <string name="app_info_text">App-ontwikkeling: \n<b><a href="http://www.gaby-becker.de">Gaby Becker</a></b>
+        \nin 08/2016 tijdens het opensourceproject van <b><a href="http://www.deutschlands-bahnhoefe.org">Deutschlands Bahnhöfe</a></b>.          \n\nDe stationsinformatie is aangeleverd door Deutsche Bahn in de context van het lokale <b><a href="http://data.deutschebahn.com/pages/nutzungsbedingungen">OpenData-project</a></b>.          \n\nHet json-bestand dat de foto\'s en stationsinformatie samenvoegt is aangeleverd door de Java-app, ontwikkeld door <b><a href="https://twitter.com/storchp">Peter Storch</a></b>.           \n\nDe app gebruikt de <a href="https://design.google.com/icons/">material-design-icons van Google,</a> uitgebracht onder de Apache-licentie, versie 2.0\n\n<b>Medewerkers</b> van het 'Deutschlands-Bahnhöfe'-project: \n\n<b>- <a href="https://www.twitter.com/khgdrn">Kirsten Hagedor n (kaartsamenstelling)</a></b>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: - `Deutschlands BahnhÃ¶fe` -> `Deutschlands Bahnhöfe`
- `van het 'Deutschlands-BahnhÃ¶fe'-project`-> `van het \'Deutschlands-Bahnhöfe\'-project`

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        \n\n<b>- Alexandra Schön (ontwerk)</b>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: - `Alexandra SchÃ¶n` -> `Alexandra Schön`

File: app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        \n\n<b>- Katrin Schönherr (php-ontwikkeling)</b>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: `Katrin SchÃ¶nherr` -> `Katrin Schönherr`

File: app/src/main/res/values-nl/strings.xml [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+    </string>
+  <string name="map_info_title">Over kaarten</string>
+  <string name="map_info_ok">Ok�</string>
+  <string name="map_info_text">De standaardkaart is aangeleverd door <b><a href="https://openstreetmap.org">OpenStreetMap.org</a></b>.
+        \nWordt lid van de gemeenschap en help de kaart te verbeteren. Zie <b><a href="https://www.openstreetmap.org/fixthemap">www.openstreetmap.org/fixthemap</a></b>
+        \n\nOverweeg een offline-kaart te downloaden om de serverbelasting te verlagen en mb's te besparen.
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: `en mb's te besparen` -> `en mb\'s te besparen`

File: app/src/main/res/values-nl/strings.xml [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+  <string name="menu_title_stations_with_photo">Stations voorzien van een foto</string>
+  <string name="menu_title_stations_without_photo">In de omgeving</string>
+  <string name="menu_title_all_stations_without_photo">Alle stations zonder foto</string>
+  <string name="menu_title_share">Delen</string>
+  <string name="menu_title_send">Versturen</string>
+  <string name="action_settings">Instellingen</string>
+  <string name="maps_activity">Stationskaart</string>
+  <string name="no_location_detected">Kan locatie niet ophalen</string>
+  <string name="title_activity_details">Stationsinformatie</string>
+  <string name="no_stations_loaded">Geen stations geladen; druk eerst op 'Bijwerken'.</string>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: `'Bijwerken'` -> `\'Bijwerken\'`

File: app/src/main/res/values-nl/strings.xml [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+  <string name="text_intro_slider2">Voer de vereiste gegevens in onder \'Mijn gegevens\' zodat we je fotobestanden van de juiste naam kunnen voorzien. Zonder bijnaam kun je geen foto\'s plaatsen. \n\n<b>Je kunt de upload-toegangssleutel snel en makkelijk ophalen.</b></string>
+  <string name="text_intro_slider3">In het menu onder<b> \'Stationskaart\' </b>en op de <b>lijst op de startpagina</b> vind je een keuze om <b>alle</b> stations te laden (duurt langer), alleen de stations <b>voorzien</b> van een foto, of alleen stations <b>zonder</b> foto.</string>
+  <string name="text_intro_slider4">Er zijn twee manieren om een foto van een station te maken: middels het zoekvak op de <b>startpagina</b> of <b>middels het menuitem \'Stationskaart\'</b>. Het fotopictogram is alleen aanklikbaar als je je telefoon horizontaal houdt.</string>
+  <string name="text_intro_slider5">Zodra je een foto hebt genomen, wordt deze getoond en kun je hem uploaden.<b>Niet-bestaande stations kunnen worden gemeld middels de knop 'Nieuw'.</b></string>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: `'Nieuw'` -> `\'Nieuw\'`

File: app/src/main/res/values-nl/strings.xml [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+  </plurals>
+    <string name="openWebsite">Website openen</string>
+  <string name="button_ok_text">Ok�</string>
+    <string name="confirm_replace_with_ghost">Wil je de huidige foto vervangen door een spookstation?</string>
+    <string name="upload_bad_request">Onjuist verzoek (onbekend station, onjuist bestandsformaat, ongeldige gebruiker, ?)</string>
+  <string name="upload_too_big">De foto is te groot (max. 20 MB)</string>
+    <string name="anonymous">Anoniem</string>
+  <string name="profile_wrong_data">Onjuist verzoek: er is iets mis met je gegevens. Neem contact op met ons ondersteuningsteam.</string>
+  <string name="profile_conflict">Er is een conflict met je gebruikersnaam en/of e-mailadres. Neem contact op met ons ondersteuningsteam.</string>
+    <string name="yesPhotoOwner">Ja, dit zijn mijn eigen foto's</string>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: `foto's` -> `foto\'s`

**Comment by Vistaus** [comment]
Ah, I see. I forgot to escape a few apostrophes. But I don't know what happened to the umlauts; git must've messed those up during git push! But thanks for correcting, though! :)

**Comment by Vistaus** [comment]
Done! All should be good now! :)

File: app/src/main/res/values-nl/strings.xml [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+  <string name="count_of_images">Je hebt %1$d stationsfoto(\'s)</string>
+  <string name="search_hint">Station zoeken</string>
+  <string name="app_info_title">Over de app</string>
+  <string name="app_info_ok">Oké</string>
+  <string name="app_info_text">App-ontwikkeling: \n<b><a href="http://www.gaby-becker.de">Gaby Becker</a></b>
+        \nin 08/2016 tijdens het opensourceproject van <b><a href="http://www.deutschlands-bahnhoefe.org">Deutschlands Bahnhöfe</a></b>.          \n\nDe stationsinformatie is aangeleverd door Deutsche Bahn in de context van het lokale <b><a href="http://data.deutschebahn.com/pages/nutzungsbedingungen">OpenData-project</a></b>.          \n\nHet json-bestand dat de foto\'s en stationsinformatie samenvoegt is aangeleverd door de Java-app, ontwikkeld door <b><a href="https://twitter.com/storchp">Peter Storch</a></b>.           \n\nDe app gebruikt de <a href="https://design.google.com/icons/">material-design-icons van Google,</a> uitgebracht onder de Apache-licentie, versie 2.0\n\n<b>Medewerkers</b> van het 'Deutschlands-BahnhÃ¶fe'-project: \n\n<b>- <a href="https://www.twitter.com/khgdrn">Kirsten Hagedor n (kaartsamenstelling)</a></b>
```
Comment (pstorch) [inline app/src/main/res/values-nl/strings.xml]: One issue missing:
- `'Deutschlands-BahnhÃ¶fe'` -> `\'Deutschlands-Bahnhöfe\'`

Comment (Vistaus) [reply app/src/main/res/values-nl/strings.xml]: Not anymore :wink:

**Comment by pstorch** [comment]
Build is green 🎉, thanks for your quick response.

**Comment by Vistaus** [comment]
You're welcome, and thanks again for helping me fix the few issues that arose :)

**Pull request merged**